django_netjsonconfig/static/django-netjsonconfig/js/unsaved_changes.js

```diff
@@ -27,28 +29,29 @@
                     try{
                         object[name] = JSON.parse(value);
                     }
-                    catch(e){}
+                    catch(ignore){}
                 }
             });
         };
 
     var unsaved_changes = function(e) {
         // get current values
-        current_values = {}
+        var current_values = {};
         map_values(current_values);
         var changed = false,
             message = 'You haven\'t saved your changes yet!',
-            initialField, initialValue, currentValue;
+            initialField, initialValue, //currentValue,
+            name;
         if (gettext) { message = gettext(message); }  // i18n if enabled
         // compare initial with current values
         for (name in django._njc_initial_values) {
             // use initial values from initial fields if present
             initialField = $('#initial-id_' + name);
             initialValue = initialField.length ? initialField.val() : django._njc_initial_values[name];
             // fix checkbox value inconsistency
-            if (initialValue == 'True') { initialValue = true }
-            else if (initialValue == 'False') { initialValue = false }
-            if (name == 'config') { initialValue = JSON.parse(initialValue) }
+            if (initialValue == 'True') { initialValue = true; }
+            else if (initialValue == 'False') { initialValue = false; }
+            if (name == 'config') { initialValue = JSON.parse(initialValue); }
             // currentValue = current_values[name]
```

Comment:
mm `currentValue` looks like dead forgotten code, you may remove all occurrences of it if it's true